sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -82,7 +82,9 @@ singleTableSchema
 statement
     : query                                                            #statementDefault
     | ctes? dmlStatementNoWith                                         #dmlStatement
-    | USE db=errorCapturingIdentifier                                  #use
+    | USE CATALOG catalog=errorCapturingIdentifier                     #useCatalog
+    | USE namespace=multipartIdentifier
```

Comment:
shall we add a command to show the current catalog and name space? e.g. `SHOW CURRENT CATALOG`, `SHOW CURRENT NAMESPACE`

cc @rdblue @brkyvz

Reply:
+1. yea I think that will be useful.